data/json/recipes/recipe_food.json

```diff
@@ -6674,9 +6679,10 @@
     "autolearn": true,
     "batch_time_factors": [ 50, 3 ],
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 1 } ],
+    "tools": [ [ [ "funnel", -1] ] ],
```

Comment:
```suggestion
    "tools": [ [ [ "funnel", -1 ] ] ],
```

Reply:
Shouldn't it allow to use makeshift funnel too?